config.go

```diff
@@ -45,7 +45,7 @@ type Root struct {
 type Platform struct {
 	// OS is the operating system.
 	OS string `json:"os"`
-	// Arch is the architecture
+	// Arch is the architecture.
```

Comment:
the periods at the end of this sentence are not required. This terse of a comment can be regarded as the same rules as a bullet list.